index.js

```diff
@@ -25,14 +25,18 @@ module.exports = function(opt) {
         printWidth: 80,
         // Number of spaces it should use per tab
         tabWidth: 2,
-        // Use the flow parser instead of babylon
-        useFlowParser: true,
+        // Use tabs instead of spaces
+        useTabs: false,
+        // Remove semicolons
+        semi: true,
```

Comment:
doesn't `semi: true` mean to *keep* semicolons?  If that's the case, the comment above should be updated to `// Keep semicolons`